apps/ae_socket_connector/lib/channel_runner.ex

```diff
@@ -7,7 +7,8 @@ defmodule ChannelRunner do
   @pause 2000
 
   def start_channel_helper(),
-    do: start_channel_helper(:single)
+    # do: start_channel_helper(:single)
+    do: ClientRunner.start_helper()
```

Comment:
`start_channel_helper/0` and `start_channel_helper/1` seem to have very different semantics now. Should they have different function names?